internal/service/pushbasedlogexport/resource.go

```diff
@@ -51,7 +52,15 @@ func (r *pushBasedLogExportRS) Create(ctx context.Context, req resource.CreateRe
 	connV2 := r.Client.AtlasV2
 	projectID := tfPlan.ProjectID.ValueString()
 	if _, err := connV2.PushBasedLogExportApi.CreatePushBasedLogConfiguration(ctx, projectID, logExportConfigReq).Execute(); err != nil {
-		resp.Diagnostics.AddError("eError when creating push-based log export configuration", err.Error())
+		resp.Diagnostics.AddError("Error when creating push-based log export configuration", err.Error())
+
+		log.Printf("[INFO] Unconfiguring push-based log export for project due to create failure: %s", projectID)
```

Comment:
I think this should be part of the `AddError` message (eventually)

Reply:
Could be, but I chose to log this separately so it's understood why we need to make this API call